tests/test_models.py

```diff
@@ -51,6 +53,28 @@ def test_active_customers_get_queryset_returns_only_active(self):
 
 @mark.django_db
 @ddt.ddt
+class TestUserDataSharingConsentAudit(unittest.TestCase):
+    """
+    Tests of the UserDataSharingConsent model.
+    """
+    @ddt.data(
+        str, repr
+    )
+    def test_string_conversion(self, method):
+        """
+        Test ``UserDataSharingConsentAudit`` conversion to string
+        """
+        audit = UserDataSharingConsentAuditFactory()
+        expected_to_str = "<{class_name} for {user}: {state}>".format(
+            class_name=UserDataSharingConsentAudit.__name__,
+            user=audit.user,
+            state=audit.state
+        )
```

Comment:
Nit: This test code is almost identical to the implementation code. IMHO it would be better to construct an `UserDataSharingConsentAudit` instance with a known user name and then just compere the result of `method(audit)` to a hard-coded string like `"<UserDataSharingConsent for Bob: Not set>"`. Then it's also clear, for example, if `state` is the verbose version or the internal value.